basic-javascript/project-1.spec.js

```diff
@@ -177,3 +177,58 @@ describe('isOdd', () => {
     expect(isOdd(num)).toBeFalsy()
   })
 })
+
+describe('square', () => {
+  test('test whether square() returns square', () => {
+    const num = 9
+    const expected = num * num
```

Comment:
Although this works, I would suggest hard coding the value for your expected output. The way you are currently testing it, is pretty much the implementation of the square function :)